examples/ReferenceTestV2/reference_consumer_v2.py

```diff
@@ -770,3 +763,6 @@ def on_operational_state_updates(operational_states_by_handle):
 
     run_ref_test(results)
     results.print_summary()
+    if results.failed_count:
+        exit(-1)

```

Comment:
its not a good idea to use negative exit codes as each os will handle them differently (e.g. linux uses unsigned exit codes). I suggest to just use `1` as exit code. I suggest to change the if clause to `exit(bool(results.failed_count))`

Reply:
exit code changed.